templates/masterclasses.html

```diff
@@ -52,6 +52,14 @@ <h3 class="p-media-object__title">
                   <li class="p-media-object__meta-list-item--date">
                     <span class="u-off-screen">Date: </span>{{ session.Date.Formatted }}
                   </li>
+                  <!-- Add to calendar button -->
+                  <!-- Currently, all events start at 11AM CST with duration of 45 minutes -->
+                  <li class="p-media-object__meta-list-item--date">
```

Comment:
Using the same icon twice for the date and the add to calendar looks a bit odd, maybe we can find a better one or avoid the logo for the add to calendar row?